components/product/AddToCartButton/vtex.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCart } from "apps/vtex/hooks/useCart.ts";
+import Button, { Props as BtnProps } from "./common.tsx";
+import { AnatomyClasses, handleClasses } from "../../../sdk/styles.ts";
+
+
+const anatomy = [
+  "container",
+] as const;
+type Styles = AnatomyClasses<typeof anatomy[number]>;
+
+export interface Props extends Omit<BtnProps, "onAddItem"> {
+  seller: string;
+  productID: string;
+  classes?: Styles;
+  textCart?:string;
+}
+
+function AddToCartButton({ seller, productID, eventParams,classes,textCart }: Props) {
+  const { addItems } = useCart();
+  const onAddItem = () =>
+    addItems({
+      orderItems: [{
+        id: productID,
+        seller: seller,
+        quantity: 1,
+      }],
+    });
```

Comment:
Vamos dar preferência para usar `function` sempre que possível, tirando quando for callback de alguma outra função (map, reduce, etc)

```suggestion
  function onAddItem() {
    addItems({
      orderItems: [{
        id: productID,
        seller: seller,
        quantity: 1,
      }],
    });
 }
```

Reply:
Ok, troca foi feita.